### PR TITLE
Use typescript@4.3 and ts-node@10.1

### PR DESCRIPTION
**Pull request by jozefizso**

### CODE REVIEW

File: packages/custom-functions-metadata/src/generate.ts [hunk 1]
```diff
@@ -739,9 +739,9 @@ function findTag(node: ts.Node, tagName: string): ts.JSDocTag | undefined {
 /**
  * If a node contains the named tag, returns the tag comment, otherwise returns "".
  */
-function getTagComment(node: ts.Node, tagName: string) {
+function getTagComment(node: ts.Node, tagName: string): string {
   const tag = findTag(node, tagName);
-  return tag && tag.comment ? tag.comment : "";
```
Comment (akrantz) [inline packages/custom-functions-metadata/src/generate.ts:744]: Could this be updated to `return tag?.comment?.toString() || ""` ?  Do you think it is better that way?

**Review by akrantz — APPROVED** [review]
Thanks for the PR! This looks good. I made one suggestion about a code change.

**Review by akrantz — CHANGES_REQUESTED** [review]
Seems like it's better for me to mark this as request changes. Mention me if you make an update or if you don't want to, and I can follow up then.

**Comment by jozefizso** [comment]
@akrantz I change the code as requested.

**Review by akrantz — APPROVED** [review]

**Pull request merged**